src/components/commands.js

```diff
@@ -0,0 +1,36 @@
+/**
+ * Index all commands
+ * The name of the file will be used as the command name
+ * Commands can be multiple directories deep
+ * @type {Object}
+ */
+
+const dirreq = require('../internal/dir-require')
+const Command = require('../classes/Command')
+const commands = dirreq('src/commands/**/*.js')
+const final = {
+  commands: {},
+  aliases: new Map()
+}
+
+Object.keys(commands).forEach(x => {
+  /* eslint-disable no-throw-literal */
+  try {
+    if (!(commands[x] instanceof Command)) throw `Command ${x} is not a WildBeast command, skipping`
+    if (final.commands[x]) throw `Can't index command ${x}, this command is duplicated, skipping`
```

Comment:
This error message could be a bit more declarative:
```suggestion
    if (final.commands[x]) throw `Command ${x} is a duplicate, skipping`
```